crates/builder/src/tasks/block.rs

```diff
@@ -131,3 +188,81 @@ impl BlockBuilder {
         (sender, handle)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::config::BuilderConfig;
+    use std::str::FromStr;
+
+    use alloy_consensus::{SignableTransaction, TxEip1559};
+    use alloy_primitives::{bytes, Address, TxKind, U256};
+    use alloy_signer::SignerSync;
+    use alloy_signer_local::PrivateKeySigner;
+
+    #[ignore = "integration test"]
```

Comment:
this will cause it to be ignored even if the feature is enabled, i think

Reply:
why do we also want to ignore it too? imo this should run in CI

Reply:
Yes, this was done intentionally as CI is not setup to test this yet. It needs env configs and possibly some permissions before it can run in CI.

Reply:
Cool, I'll move it and make a note about why it's ignored for now.